docs/docs/docs_es/introduction/motivation.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Motivación
+---
+
+Por razones de compatibilidad y simplicidad, es mejor usar las funciones nativas de React para el manejo de estados, en lugar de utilizar estados externos y globales. Pero React tiene algunas limitantes:
+
+- El estado de un componente solo puede ser compartido, si éste estado se mueve hacia el componente padre que se tiene en común, pero este implica tener un árbol de componentes mas grande que renderizar.
+- El contexto de React solo puede mantener un valor, y no un conjunto indefinido de valores con sus respectivos consumidores.
+- Por estas dos razones, es difícil dividir el código de la parte más alta del árbol de componentes (donde tenemos el estado), de las hojas del mismo árbol (donde el estado es usado).
+
+Queremos mejorar esto y al mismo tiempo seguir con la misma API, semántica y comportamiento de React como sea posible.
+
+Recoil define un grafo ortogonal directo pero también intrínseco a tu árbol de componentes de React. Los cambios de estado fluyen desde las raíces de este grafo (que llamamos átomos), pasan a través de funciones puras (que hemos llamado selectores) y finalmente llegan hasta los componentes.
+Con este enfoque:
+
+- Crear una API libre de código repetitivo, donde el estado ha compartir pueda utilizar la misma interfaz get/set, igual como lo hace el estado interno de React (Aunque también se puede encapsular con reductores de ser necesario).
+- Tenemos la posibilidad de ser compatibles con el Modo Concurrente y cualquier otra funcionalidad nueva de React que este disponible.
+- La definición del estado es incremental y distribuido, permitiendo así dividir el código.
+- El estado puede ser reemplazado con datos derivados sin tener que modificar el component que lo usa.
+- Los datos derivados pueden ser cambiados para ser asíncronos o síncronos, sin tener que modificar el component que lo usa.
+- Podemos usar la navegación como concepto de primera clase, incluso codificar el estado en hipervínculos.
+- Es fácil mantener todo el estado de la aplicación persistentemente y compatible con versiones anteriores, por lo tanto éste estado persistente puede sobrevivir cambios en la aplicación.
```

Comment:
por lo tanto-> por lo que puede sobrevivir cambios en la aplicación. (remove repetition of subject)